src/pudl/transform/ferc1.py

```diff
@@ -3712,9 +3731,35 @@ class OtherRegulatoryLiabilitiesFerc1TableTransformer(Ferc1AbstractTableTransfor
     has_unique_record_ids = False
 
 
+FERC1_TFR_CLASSES: Mapping[str, type[Ferc1AbstractTableTransformer]] = {
+    "fuel_ferc1": FuelFerc1TableTransformer,
+    "plants_small_ferc1": PlantsSmallFerc1TableTransformer,
+    "plants_hydro_ferc1": PlantsHydroFerc1TableTransformer,
+    "plant_in_service_ferc1": PlantInServiceFerc1TableTransformer,
+    "plants_pumped_storage_ferc1": PlantsPumpedStorageFerc1TableTransformer,
+    "transmission_statistics_ferc1": TransmissionStatisticsFerc1TableTransformer,
+    "purchased_power_ferc1": PurchasedPowerFerc1TableTransformer,
+    "electric_energy_sources_ferc1": ElectricEnergySourcesFerc1TableTransformer,
+    "electric_energy_dispositions_ferc1": ElectricEnergyDispositionsFerc1TableTransformer,
+    "utility_plant_summary_ferc1": UtilityPlantSummaryFerc1TableTransformer,
+    "electric_operating_expenses_ferc1": ElectricOperatingExpensesFerc1TableTransformer,
+    "balance_sheet_liabilities_ferc1": BalanceSheetLiabilitiesFerc1TableTransformer,
+    "depreciation_amortization_summary_ferc1": DepreciationAmortizationSummaryFerc1TableTransformer,
+    "balance_sheet_assets_ferc1": BalanceSheetAssetsFerc1TableTransformer,
+    "income_statement_ferc1": IncomeStatementFerc1TableTransformer,
+    "electric_plant_depreciation_changes_ferc1": ElectricPlantDepreciationChangesFerc1TableTransformer,
+    "electric_plant_depreciation_functional_ferc1": ElectricPlantDepreciationFunctionalFerc1TableTransformer,
+    "retained_earnings_ferc1": RetainedEarningsFerc1TableTransformer,
+    "electric_operating_revenues_ferc1": ElectricOperatingRevenuesFerc1TableTransformer,
+    "cash_flow_ferc1": CashFlowFerc1TableTransformer,
+    "electricity_sales_by_rate_schedule_ferc1": ElectricitySalesByRateScheduleFerc1TableTransformer,
+    "other_regulatory_liabilities_ferc1": OtherRegulatoryLiabilitiesFerc1TableTransformer,
+}
```

Comment:
moved these to be a global variable so we can access each table's verified params and xbrl meta outside of the transform step/with only the table name. used down below for translating the calculations in the metadata. this translation needs to know about all of the tables.